src/SignalRServiceExtension/Config/SignalRConfigProvider.cs

```diff
@@ -72,14 +73,18 @@ public void Initialize(ExtensionConfigContext context)
             var triggerBindingRule = context.AddBindingRule<SignalRTriggerAttribute>();
             triggerBindingRule.AddConverter<InvocationContext, JObject>(JObject.FromObject);
             triggerBindingRule.BindToTrigger<InvocationContext>(new SignalRTriggerBindingProvider(_dispatcher, nameResolver, serviceManagerStore, webhookException));
-                        
+
             // Non-trigger binding rule
             var signalRConnectionInfoAttributeRule = context.AddBindingRule<SignalRConnectionInfoAttribute>();
             signalRConnectionInfoAttributeRule.Bind(inputBindingProvider);
 
             var securityTokenValidationAttributeRule = context.AddBindingRule<SecurityTokenValidationAttribute>();
             securityTokenValidationAttributeRule.Bind(inputBindingProvider);
 
+            _ = context.AddBindingRule<SignalREndpointsAttribute>()
+                   .AddConverter<LiteServiceEndpoint[], JArray>(endpoints => JArray.FromObject(endpoints))
```

Comment:
>.AddConverter<LiteServiceEndpoint[], JArray>(endpoints => JArray.FromObject(endpoints)) [](start = 19, length = 87)

can simplify to `.AddConverter<LiteServiceEndpoint[], JArray>(JArray.FromObject)....;`, maybe you can simplyify previous binding rules as well for code consistency.